torchgeo/datasets/usavars.py

```diff
@@ -165,6 +165,9 @@ def __getitem__(self, index: int) -> Dict[str, Tensor]:
                 [self.label_dfs[lab].loc[id_][lab] for lab in self.labels]
             ),
             "image": self._load_image(os.path.join(self.root, "uar", tif_file)),
+            "centroid_loc": Tensor(
+                self.label_dfs[self.labels[0]].loc[id_][["lon", "lat"]]
```

Comment:
I feel like lat/lon is more common than lon/lat. Should we call the key centroid_log or centroid?

Reply:
fiona returns lon/lats (I think this is standard in geospatial libraries)
E.g.

![image](https://user-images.githubusercontent.com/1088078/231938050-06e400ec-9088-4a95-8263-838e8dc13294.png)

When we write coordinates we usually put latitude first though, e.g. 47.661435 N, -122.116502 E.

Reply:
I'll just return lat and lon separately so that it's as clear as possible - thoughts/objections?

Reply:
That's fine too. The current solution is fine as long as we document it.